world/boards/templatetags/boards_tags.py

```diff
@@ -7,10 +7,23 @@
 def render_board_summary(board):
     return { "board": board }
 
-@register.inclusion_tag("boards/post_summary.html")
-def render_post_summary(post):
-    return { "post": post }
+@register.inclusion_tag("boards/post_list.html")
+def render_post_list(sorted_posts, focused_post):
+    return { "sorted_posts": sorted_posts, "focused_post": focused_post }
+
+@register.inclusion_tag("boards/post_nav.html")
+def render_post_nav(post, previous, next):
+    return { "post": post, "previous": previous, "next": next}
+
+@register.inclusion_tag("boards/post_detail.html")
+def render_post_detail(post, body_paragraphs):
+    return { "post": post, "body_paragraphs": body_paragraphs }
+
+@register.simple_tag
+def get_board_detail_url(board, post=None):
+    return reverse('boards:detail_for_post',
+                   kwargs={'board_id': board.id, 'post_id': post.id if post else None})
 
 @register.simple_tag
```

Comment:
It looks like the repeated code issue I just commented on in the previous PR has been fixed here. So, I have a question regarding procedure: should we pull the PRs in order, so the code is fixed through this PR, or should we not pull parts of the previous PR's code?

Reply:
If I understand your question correctly, this is specifically for the state that we WERE in a year ago 🤦 where I'd issued two chained PRs back-to-back and they were BOTH still in review.

In that case what we would do is we would notice the repeated code in the first PR and fix it. Because the second PR is branched from the first PR, I would (immediately, while everything is still in review) rebase the second PR again from the first PR which would pick up that change and the issue would be gone in both PRs.

Does that make sense? In other words, when you have code review comments/changes required on a PR that has a chain of subsequent PRs branched off of it, you would normally propagate the fixes through the entire chain before merging any of them.